### ISSUE TITLE
Error: Cannot find module 'lcov-total'

### ISSUE THREAD

**Issue by bunin** [zgosalvez/github-actions-report-lcov#61]
Tried to switch from @v1 to @v3 and got this:

```
Run zgosalvez/github-actions-report-lcov@v3
  with:
    coverage-files: coverage.lcov
    artifact-name: code-coverage-report
    github-token: ***
    minimum-coverage: 0
  env:
...
Error: Cannot find module 'lcov-total'
Require stack:
- /home/runner/work/_actions/zgosalvez/github-actions-report-lcov/v3/dist/main/index.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.2388 (/home/runner/work/_actions/zgosalvez/github-actions-report-lcov/v3/dist/webpack:/github-actions-report-lcov/node_modules/@vercel/ncc/dist/ncc/@@notfound.js:1:1)
    at __nccwpck_require__ (/home/runner/work/_actions/zgosalvez/github-actions-report-lcov/v3/dist/webpack:/github-actions-report-lcov/webpack/bootstrap:21:1)
    at /home/runner/work/_actions/zgosalvez/github-actions-report-lcov/v3/dist/webpack:/github-actions-report-lcov/src/main.js:6:1
    at /home/runner/work/_actions/zgosalvez/github-actions-report-lcov/v3/dist/main/index.js:20541:3
    at Object.<anonymous> (/home/runner/work/_actions/zgosalvez/github-actions-report-lcov/v3/dist/main/index.js:20544:12)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
```

the workflow file looks like this:

```
    runs-on: ubuntu-latest
    env:
      ...
    steps:
      - uses: actions/checkout@v3
      - name: Set up Go
        uses: actions/setup-go@v3
        with:
          go-version: 1.19
...
      - name: Setup LCOV
        uses: hrishikesh-kadam/setup-lcov@v1

      - name: Run tests
        run: go test -cover -coverprofile=gotest.coverage ./...

      - name: Convert coverage to lcov
        uses: jandelgado/gcov2lcov-action@v1
        with:
          infile: gotest.coverage
...
      - name: Report code coverage
        uses: zgosalvez/github-actions-report-lcov@v3
        with:
          coverage-files: coverage.lcov
          artifact-name: code-coverage-report
          github-token: ${{ secrets.GITHUB_TOKEN }}
```

@v1 and @v2 works fine with the same configuration.

**Comment by davestimpert** [comment]
I tried adding `- run: npm install -g lcov-total` to my GH action but it didn't help

**Comment by sagargugwad** [comment]
Same for me with @v3

**Comment by bartosz347** [comment]
Lcov-total has switched to pure ESM in their recent update and probably that's why it stopped after upgrading the package in https://github.com/zgosalvez/github-actions-report-lcov/pull/53.

I also discovered that after merging https://github.com/zgosalvez/github-actions-report-lcov/pull/52 it is necessary to adjust its usage in this project.

For reference [here's my fork branch with fixes](https://github.com/bartosz347/github-actions-report-lcov/commits/fork-main). I've reverted to previous version of `lcov-total` and adjusted to changes in GitHub helper.

I'm would be happy to open a PR here, but first would like to have https://github.com/zgosalvez/github-actions-report-lcov/pull/42 merged.

**Comment by sagargugwad** [comment]
@bartosz347 #42 is not passing the the build test check

**Comment by bartosz347** [comment]
> #42 is not passing the the build test check

@sagargugwad the workflow is [failing on repo checkout step](https://github.com/zgosalvez/github-actions-report-lcov/actions/runs/3988363092/jobs/6839419283), I believe something's wrong with the pipeline when running forks.

**Comment by zgosalvez** [comment]
This should be resolved in the latest 3.1.0 release. Thank you @bartosz347!
